rabix-tests/src/main/java/org/rabix/tests/TestRunner.java

```diff
@@ -0,0 +1,212 @@
+    resultFileName = resultValues.get("path").toString();
+    resultFileName = resultFileName.split("/")[resultFileName.split("/").length - 1];
+    resultFileSize = (int) resultValues.get("size");
+    resultFileClass = resultValues.get("class").toString();
+    logger.info("Test validation:");
+    logger.info("result file name: " + resultFileName + ", expected file name: "
+        + mapTest.get("expected").get("outfile").get("name"));
+    logger.info("result file size: " + resultFileSize + ", expected file size: "
+        + mapTest.get("expected").get("outfile").get("size"));
+    logger.info("result file class: " + resultFileClass + ", expected file class: "
+        + mapTest.get("expected").get("outfile").get("class"));
+
+    boolean fileNamesEqual = resultFileName.equals(mapTest.get("expected").get("outfile").get("name"));
+    boolean fileSizesEqual = resultFileSize == (int) mapTest.get("expected").get("outfile").get("size");
+    boolean fileClassesEqual = resultFileClass.equals(mapTest.get("expected").get("outfile").get("class"));
+
+    if (!fileNamesEqual) {
+      logger.info("result and expected file name are not equal!");
+    } else {
+      if (!fileSizesEqual) {
+        logger.info("result and expected file size are not equal!");
+      } else {
+        if (!fileClassesEqual) {
+          logger.info("result and expected file class are not equal!");
+        } else {
+          return true;
+        }
+      }
+    }
+
+    return false;
+  }
+
+  public static ArrayList<String> command(final String cmdline, final String directory) {
+    try {
+      Process process = new ProcessBuilder(new String[] { "bash", "-c", cmdline }).redirectErrorStream(true)
+          .directory(new File(directory)).start();
+
+      ArrayList<String> output = new ArrayList<String>();
+      BufferedReader br = new BufferedReader(new InputStreamReader(process.getInputStream()));
+      String line = null;
+      while ((line = br.readLine()) != null)
+        output.add(line);
+
+      if (0 != process.waitFor()) {
+        return null;
+      }
+
+      return output;
+
+    } catch (Exception e) {
+      return null;
+    }
+  }
+
+  static void executeCommand(String cmdline) {
+    ArrayList<String> output = command(cmdline, workingdir);
+    if (null == output)
+      logger.info("COMMAND FAILED: " + cmdline + "\n");
+    else
+      for (String line : output) {
+        logger.info(line);
+      }
+  }
+
+  /**
+   * Reads content from a file
+   */
+  static String readFile(String path, Charset encoding) throws IOException {
+    byte[] encoded = Files.readAllBytes(Paths.get(path));
+    return new String(encoded, encoding);
+  }
+
+  private static String getConfig(String key) throws IOException {
+    File configFile = new File(System.getProperty("user.dir") + "/rabix-tests/config/core.properties");
+    String config = readFile(configFile.getAbsolutePath(), Charset.defaultCharset());
+    String[] splitedRows = config.split("\n");
```

Comment:
use **Apache commons configuration** for handling property files

PropertiesConfiguration configuration = new PropertiesConfiguration();
configuration.load(configFile);